src/db/mod.rs

```diff
@@ -443,19 +422,18 @@ impl ThreadedDatabase for Database {
                  users: Vec<&str>,
                  options: Option<UserInfoOptions>)
                  -> Result<Vec<bson::Document>> {
-        let info_options = options.unwrap_or_else(UserInfoOptions::new);
-        let vec = users.into_iter()
+        let vec: Vec<_> = users.into_iter()
             .map(|user| {
-                let doc = doc! { "user" => user, "db" => (Bson::String(self.name.to_owned())) };
+                let doc = doc! { "user" => user, "db" => (self.name.to_owned()) };
```

Comment:
No need to specify the enum variant here, as the `From` trait handles that for us.